### PR TITLE
#9 Backend for adding, viewing and voting racks

### PR DESCRIPTION
**Pull request by martarozek**
I branched from facebook-login because I needed the User model

### CODE REVIEW

File: fietsenrek/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  {% block head %} {% endblock %}
+</head>
+<body>
+    {% block content %} {% endblock %}
+</body>
+</html>
```
Comment (GraczykowskiMichal) [inline fietsenrek/templates/base.html:12]: I don't think we need any templates if we are developing separate frontend app

File: fietsenrek/templates/home.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h1>FIETSENREK HOMEPAGE</h1>
+{% endblock content %}
```
Comment (GraczykowskiMichal) [inline fietsenrek/templates/home.html:5]: Same as above

File: fietsenrek/templates/base.html (hunk 1 above)
Comment (martarozek) [reply fietsenrek/templates/base.html:12]: You're right and I'll have to look at the authentication again as I used a library that takes care of it all and doesn't provide an API.

Comment (GraczykowskiMichal) [reply fietsenrek/templates/base.html:12]: Oki

**Pull request merged**